src/azure-cli/azure/cli/command_modules/sql/_help.py

```diff
@@ -845,9 +845,9 @@
     text: az sql mi update --name myinstance --public-data-endpoint-enabled true --resource-group mygroup --subscription MySubscription
     crafted: true
   - name: Update a managed instance with maintenance configuration
-    text: az sql mi update -g mygroup  -n myinstance -m /subscriptions/{SubID}/providers/Microsoft.Maintenance/publicMaintenanceConfigurations/SQL_{Region}_{MaintenanceConfigName}
```

Comment:
Do you still support previous long name with resource id?

Reply:
Yes